testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-ldap-bind-dn/password-authenticator.properties

```diff
@@ -1,7 +1,7 @@
 password-authenticator.name=ldap
 ldap.url=ldaps://ldapserver:636
 ldap.ssl-trust-certificate=/etc/openldap/certs/openldap-certificate.pem
-ldap.user-base-dn=ou=Asia,dc=presto,dc=testldap,dc=com
+ldap.user-base-dn=dc=presto,dc=testldap,dc=com
```

Comment:
I've added `uid=EuropeUser,ou=Europe` and `uid=AmericanUser,ou=America` who are a memberOf `DefaultGroup` so both of them should be able to run queries provided they can bind to LDAP.

I've changed the `ldap.user-bind-pattern` to include both `ou=America` and `ou=Asia` so that every user except `EuropeUser` can bind. And base-dn has been changed to remove `ou=Asia` because we want to search every `ou`.